src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/doc_viewer_span_overview/span_overview.tsx

```diff
@@ -71,18 +70,13 @@ export function SpanOverview({
               <SpanDurationSummary duration={spanDuration} />
             </>
           )}
-          {transactionId && (
```

Comment:
Q: why there is no need to check for the `parsedDoc[SPAN_ID_FIELD]` while in line https://github.com/elastic/kibana/pull/218599/files#diff-11dd0890f96c58891f2d69b7ae46660802f5852a9f9975a9de600907740f206cR81 we do check?